tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -116,8 +107,74 @@ def test_createami_post_install(
         + post_install_args
     )
 
-    assert_that(
-        any("No post install script".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
-    ).is_false()
+    stdout_lower = pcluster_createami_result.stdout.lower()
+    assert_that(stdout_lower).does_not_contain("no post install script")
+    assert_that(stdout_lower).does_not_contain("no custom ami created")
+
+
+@pytest.mark.dimensions("eu-central-1", "c5.xlarge", "alinux", "sge")
+def test_createami_wrong_os(
+    region, os, instance, test_datadir, request, pcluster_config_reader, vpc_stack, architecture, scheduler
```

Comment:
`scheduler` is not used in this test, so probably don't need to pass it in?

Reply:
Done